plugin/pkg/admission/limitranger/admission_test.go

```diff
@@ -242,7 +244,7 @@ func TestMergePodResourceRequirements(t *testing.T) {
 			t.Errorf("pod %v, expected != actual; %v != %v", pod.Name, expected, actual)
 		}
 	}
-	verifyAnnotation(t, &pod, "LimitRanger plugin set: cpu request for container foo-0; cpu, memory limit for container foo-0")
+	verifyAnnotation(t, &pod, "LimitRanger plugin set: cpu request for container foo-0; cpu limit for container foo-0")
```

Comment:
Do we also want to add a test for the memory case?

Reply:
fine, will add in testcase later.

Reply:
Done @ehashman
Thanks.